phpBB/assets/javascript/core.js

```diff
@@ -407,8 +407,11 @@ phpbb.ajaxify = function(options) {
 	if (isForm) {
 		$elements.find('input:submit').click(function () {
 			var $this = $(this);
+			var $form_submits = $(this).parents('form:first').find('input:submit[data-clicked]');
```

Comment:
Use `$this` instead of `$(this)`. Chain the variables. `$form_submits` isn't very descriptive. `$clicked_buttons` maybe?